### DISCUSSION TITLE
如何设置初始化schemeField的值

### DISCUSSION THREAD

**Discussion by Amoslb** [alibaba/formily#2081] [Q&A 2.x]
抱歉，翻了好久文档一直没找到。![企业微信截图_25197c56-a5a8-4405-88f0-41373478802a](https://user-images.githubusercontent.com/22581560/131505238-2d127f10-e78b-4020-ad7c-8b975c2f3112.png)想在初始化的时候动态更改enum的值，如何做。我看文档大致![企业微信截图_cd102fa9-6884-44d2-9f6d-94259ca26f08](https://user-images.githubusercontent.com/22581560/131505580-61597e51-1fc0-475d-8808-fe83e8d6c654.png) 但是不清楚这个方法里面如何修改field 的dataSource

**Comment by Amoslb** [comment]
已解决

**Reply by MatriXiao88** [reply]
怎么解决的？
